tests/Fixer/ArrayNotation/ArraySyntaxFixerTest.php

```diff
@@ -25,9 +25,9 @@
 {
     public function testInvalidConfiguration()
     {
-        $this->setExpectedException(
+        $this->setExpectedExceptionRegExp(
             'PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException',
-            '[array_syntax] Invalid configuration: The option "a" does not exist.'
+            '#^\[array_syntax\] Invalid configuration: The option \"a\" does not exist\. Defined options are: \"syntax\"\.$#'
```

Comment:
The error message is slightly different when using `symfony/options-revolver` 2.6: the word _Defined_ is replaced with another one I don't rembember.